ncm-ccm/src/main/perl/ccm.pm

```diff
@@ -52,7 +52,11 @@ sub Configure
     delete($t->{version});
 
     while (my ($k, $v) = each(%$t)) {
-        print $fh "$k $v\n";
+        if (ref($v) eq 'ARRAY') {
+            print $fh "$k " . join(',', @$v) . "\n" unless scalar(@$v) == 0;
```

Comment:
the `scalar(@$v) == 0` is not really needed, yo can do logic tests with `unless @$v`.
but the not printing seems odd. it's not needed for the schema, and if you want this, you should also do this for regular scalars. so the above code can become
```perl
print $fh "$k $value\n" if $value;
```